src/devices/honyar.ts

```diff
@@ -128,6 +181,36 @@ const definitions: Definition[] = [
             await reporting.onOff(endpoint3);
         },
     },
+	{
+		zigbeeModel: ['000a0abb\u0000', 'RH5000_SmartOutlet'],
+		model: 'IHC8223AL',
+		vendor: 'Honyar',
+		description: 'Smart Power Socket 10A (with power monitoring)',
+		fromZigbee: [fz.on_off, fzLocal.honyer_electrical_measurement, fzLocal.honyer_metering],
```

Comment:
Could you use modernExtend instead here? Example:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/3553ddedf8cf4ae3751df924b7ea7c0babefacfd/src/devices/adeo.ts#L297

Reply:
Continue to try to modify, this time there is no error. Please help to check if there are any mistakes.

Reply:
I don't see the `extend` in this PR yet

Reply:
Can't I do it the way I'm doing it?

Reply:
No, only modernExtend is accepted since this reduces code.

Reply:
I use modernExtend for the first time, but I am not sure whether it is successful. Please help to check it.